test/functionalTest/cases/1585_log_summary/log_summary.test

```diff
@@ -25,8 +25,16 @@ Log Summary
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 0 IPv4 -logSummary 3
-
+if [ "$FUNC_TEST_RUNNING_UNDER_VALGRIND" == "true" ]
+ then
+  export BROKER_AWAIT_SLEEP_TIME=6
+  brokerStart CB 0 IPv4 -logSummary 3 
+  unset BROKER_AWAIT_SLEEP_TIME
+  sleep 7
+  valgrindSleep 8
+else 
+  brokerStart CB 0 IPv4 -logSummary 3 
+fi
 --SHELL--
```

Comment:
Strange mix here between "$FUNC_TEST_RUNNING_UNDER_VALGRIND" == "true" and calls to valgrindSleep ...

valgrindSleep does this:

    if [ "$FUNC_TEST_RUNNING_UNDER_VALGRIND" == "true" ]
    then
      sleep $1
    fi

So, if we already know that $FUNC_TEST_RUNNING_UNDER_VALGRIND is true, we can call sleep right away.

So, 

    sleep 7
    valgrindSleep 8

should be changed for 

    sleep 15

Or even better, use 

    valgrindSleep 15

after the if/else/fi.
And, as its safe to UNSET a variable, we can move around that part also ...
That way the code gets easier to read at the same time:


    if [ "$FUNC_TEST_RUNNING_UNDER_VALGRIND" == "true" ]
    then
        export BROKER_AWAIT_SLEEP_TIME=6
    fi

    brokerStart CB 0 IPv4 -logSummary 3 
    unset BROKER_AWAIT_SLEEP_TIME
    valgrindSleep 15

Same thing valid for more cases in this PR.